src/main/scala/mesosphere/marathon/core/launcher/OfferProcessorConfig.scala

```diff
@@ -9,6 +9,11 @@ trait OfferProcessorConfig extends ScallopConf {
     descr = "Offer matching timeout (ms). Stop trying to match additional tasks for this offer after this time.",
     default = Some(1000))
 
+  lazy val saveTasksToLaunchTimeout = opt[Int]("save_tasks_to_launch_timeout",
```

Comment:
This new flag has to be added to the docs and the changelog.

From this description alone, I don't really get which tasks Marathon will try to save and what would happen if it times out.